### DISCUSSION TITLE
[Enhancement] [color-picker] ColorPicker 颜色选择器 可以自定义触发器

### DISCUSSION THREAD

**Discussion by mayiyahei1992** [element-plus/element-plus#15771] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `color-picker`

## Description

ColorPicker 颜色选择器 可以自定义触发器 如普通的 Button a div 等...

### Additional comments

_(empty)_

**Comment by xxholly32** [comment]
![image](https://github.com/element-plus/element-plus/assets/6063358/94fdab05-d784-498c-9562-3cfea83bb741)

是的，这个功能感觉非常好

**Comment by tyj-321** [comment]
https://github.com/element-plus/element-plus/pull/14326

有人提过这个pr，但是并没有被merge进去，也没有更多详细信息，也不知道为什么官方不合进去

**Reply by tyj-321** [reply]
不过他这个是定义了一个trigger插槽，不是使用的默认插槽

还有他这个没有写单元测试，没有补文档啥的
